app/jobs/bulk_zombie_url_uploader_job.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class BulkZombieUrlUploaderJob < ApplicationJob
+  queue_as :searchgov
+
+  def perform(user, filename, filepath)
+    @user = user
+    @filename = filename
+    @uploader = BulkZombieUrlUploader.new(filename, filepath)
+    @uploader.upload
+    report_results
+  end
+
+  private
+
+  def report_results
+    log_results
+    send_results_email
+  end
+
+  def log_results
+    results = @uploader.results
+    Rails.logger.info "BulkZombieUrlUploaderJob: #{results.file_name}"
```

Comment:
This is not how we send extra data with our logger.  This extra spaces will just look weird in the logs. Just send extra data as a hash. Our logger will formate it accordingly.
```
 Rails.logger.info(BulkZombieUrlUploaderJob: results.file_name, total_urls: results.total_count, errors:  results.error_count)
```